### PR TITLE
Standardise domain specific variable naming - Closes #826

### LINKED ISSUE #826: Standardise domain specific variable naming
Domain specific variables should be standardised within the application code and tests.

For example, when referencing a transaction, we currently have a mix of:

```js
var transaction;
var trs;
var tx;
var t;
```

It would be better if we chose one and stuck with it throughout the whole code.

### PR DESCRIPTION
**Pull request by MaciejBaj**
Rename usages of `txs` with `transactions` / `transaction`.
Rename usages of `tx` with `transaction`. 
Changes includes code base, comments and tests.
Closes #826.

### CODE REVIEW

File: test/unit/logic/transactionPool.js [hunk 1]
```diff
@@ -72,20 +72,20 @@ describe('txPool', function () {
...
 				expect(err).to.exist;
 				done();
 			});
 		});
 
-		it('should process tx if valid and insert tx into queue', function (done) {
+		it('should process transaction if valid and insert transaction into queue', function (done) {
 			var account = node.randomAccount();
-			const tx = node.lisk.transaction.createTransaction(account.address, 100000000000, node.gAccount.password);
+			const transaction = node.lisk.transaction.createTransaction(account.address, 100000000000, node.gAccount.password);
```
Comment (SargeKhan) [inline test/unit/logic/transactionPool.js]: we are using `const`for variable decleration?

Comment (MaciejBaj) [reply test/unit/logic/transactionPool.js]: seems like 🗡

File: test/unit/logic/transaction.js [hunk 2]
```diff
@@ -1078,24 +1078,24 @@ describe('transaction', function () {
...
-			expect(trs).to.be.an('object');
-			expect(trs.asset).to.have.property('data');
+			expect(__transaction).to.be.an('object');
+			expect(__transaction.asset).to.have.property('data');
 		});
 
 		it('should return null if id field is not present', function () {
-			var rawTrs = _.cloneDeep(rawTransaction);
-			delete rawTrs.t_id;
+			var __rawTransaction = _.cloneDeep(rawTransaction);
```
Comment (LucasIsasmendi) [inline test/unit/logic/transaction.js]: Why not `rawTransactionClone`? We need to put away `__` and `_`:
```javascript
// bad
this.__firstName__ = 'Panda';
this.firstName_ = 'Panda';
this._firstName = 'Panda';

// good
this.firstName = 'Panda';
```

Comment (MaciejBaj) [reply test/unit/logic/transaction.js]: I agree.

**Pull request merged**